src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -584,14 +570,14 @@ public DescriptorImpl getDescriptor() {
          * If you don't want fields to be persisted, use <tt>transient</tt>.
          */
 
+        private boolean considerUnstableAsSuccess;
         private String credentialsId;
-        private String stashRootUrl;
+        private boolean disableInprogressNotification;
         private boolean ignoreUnverifiedSsl;
         private boolean includeBuildNumberInKey;
-        private String projectKey;
```

Comment:
@scaytrase I am wondring why `projectKey` is a field in `StashNotifier.DescriptorImpl`. It is not visible in the UI and was never part of `global.jelly`. Shouldn't it be removed from `StashNotifier.DescriptorImpl` and only be a field in `StashNotifier`?

Reply:
Not sure for it really, may be a copy-paste coincident. We can try to remove it, cannot image when project key should be declared on global level

Reply:
if it's non blocking for this PR we can try it as a separate change